### ISSUE TITLE
Alphanumerics should display units

### ISSUE THREAD

**Issue by charlesh88** [nasa/openmct#5094]
Alphanumerics should display units

### Problem
When unit metadata is available for telemetry, it should optionally be displayed wherever that value is displayed. Currently, alphanumerics in Display Layouts should display the unit, but don't. 

<img width="295" alt="image" src="https://user-images.githubusercontent.com/1056412/164089488-1f6808f6-b37a-4d73-bdbb-86341afcdb35.png">

### Solution proposal
Add a toolbar control to the right of the Value source dropdown control ("Cosine" in the shot below) that allows the user to display units or not:
![Units in layout mockup](https://user-images.githubusercontent.com/1056412/164323039-a7287cf9-3242-402e-989a-d3dd4d6db1c5.png)

- Options would be "Show units" and "Hide units", with Show being the default. This wouldn't have any effect if the user selects "Label only" from the Display Mode menu, but that's Ok. 
- The units string would be appended to the value display as shown. 
- If the user selects "Label only" the units string would not be displayed. 
- _Ideally_, selecting multiple alphanumerics while editing the layout would allow access to the Show/Hide units control so that the user can change multiples at once, but this kind of change may not be in scope as we don't do that now for other options like Display Mode.
- Also ideally, we would detect whether or not a telemetry endpoint includes units or not and only display the units dropdown control if units are available.

We need to determine if or how to migrate legacy alphanumerics; IMO all legacy alphanumerics should be set to "Hide units" to avoid breaking existing displays.

**Comment by mudinthewater** [comment]
This seems reasonable. Legacy systems would manually add text to the right of each alphanumeric.

**Comment by mudinthewater** [comment]
Presumably it would still work if the telemetry object doesn't have units in the dictionary? ie the behavior should be the same if the telemetry dictionary entry has "units": "" or just no units at all in the object.

**Comment by charlesh88** [comment]
> Presumably it would still work if the telemetry object doesn't have units in the dictionary? ie the behavior should be the same if the telemetry dictionary entry has "units": "" or just no units at all in the object.

Yes, it must fall back gracefully if no units are defined for an endpoint or if units are defined but not populated, i.e. "".
